provision/acc_provision/templates/provision-config.yaml

```diff
@@ -82,3 +82,6 @@ registry:
 
 #drop_log_config:
   # enable: True         # default is False
+
+#disable_multus:
```

Comment:
i am not sure if this section name is correct, it should be "multus" right?

Reply:
so referring to the above lines of code, this is the variable name in which the value is stored from the cmd line flag.

Reply:
This is an example of the input file, its not related to the CLI. It needs to be what is being read into the config dict, in this case you have used "multus".

Reply:
right, resolving